corporate/lib/stripe.py

```diff
@@ -1667,6 +1675,7 @@ def process_initial_upgrade(
                 # Manual license management is not available for fixed price plan.
                 assert automanage_licenses is True
                 plan_params["fixed_price"] = fixed_price_plan_offer.fixed_price
+                plan_params["end_date"] = add_months(billing_cycle_anchor, 12)
```

Comment:
Should we make this 12 be a `CustomerPlanOffer.BILLING_CYCLE_DURATION_MONTHS` that's hardcoded to 12 -- just to help clarify in the plumbing that this might in the future be a parameter? I'm not sure we can do the same for the other 12, but this seems right.

Reply:
Added in `CustomerPlan`:
```
    # Currently, all the fixed-price plans are configured for one year.
    # In future, we might change this to a field.
    FIXED_PRICE_PLAN_DURATION_MONTHS = 12
```